.travis.yml

```diff
@@ -25,27 +28,51 @@ matrix:
             - android-26
             - extra-android-m2repository
       - env: TEST_SUITE=android ABI=x86
+        language: android
+        dist: trusty
+        jdk: oraclejdk8
+        android:
+          components:
+            - android-26
+            - extra-android-m2repository 
+      # static check
+      - env: STATIC_CODE_ANALYSIS=true CHECK_C=true
```

Comment:
```
env: TEST_SUITE=static_code_analysis language:objectiveC/c++
```

Reply:
I think `TEST_SUITE=static_code_analysis` is a good advice ,but 
`language:objectiveC/c++` in travis ci has other meanings.
`CHECK_C=true` only means using oclint. maybe i should  change `CHECK_C` to `OCLint=true`